metrics_on_dashboards_report/metrics_on_dashboards_report.py

```diff
@@ -0,0 +1,93 @@
+#make sure the datadogpy library is downloaded in this machine or environment first
+from datadog import initialize,api
```

Comment:
A comment is good here but you probably shouldn't assume the user is going to read the source code beside the readme, so you can bubble this up with a `try` block like below
```
#make sure the datadogpy library is downloaded in this machine or environment first
try:
    from datadog import initialize,api
except ImportError:
    print("please make sure the datadogpy library is downloaded in this machine or environment first, find install instructions at https://github.com/DataDog/datadogpy")
    quit()
```